docs/user/vsphere/privileges.md

```diff
@@ -5,26 +5,48 @@ If the provided user has global admin privileges, no further action for permissi
 
 
 
-## New Roles
+## Create new roles with the appropriate privileges
 
 
+### Precreated virtual machine folder
+
 Role Name | vSphere object | Privilege Set
 --- | --- | ---
-
+openshift-vcenter-level | vSphere vCenter | Cns.Searchable InventoryService.Tagging.AttachTag InventoryService.Tagging.CreateCategory InventoryService.Tagging.CreateTag InventoryService.Tagging.DeleteCategory InventoryService.Tagging.DeleteTag InventoryService.Tagging.EditCategory InventoryService.Tagging.EditTag Sessions.ValidateSession StorageProfile.View
 openshift-cluster-level | vSphere vCenter Cluster | Host.Config.Storage Resource.AssignVMToPool VApp.AssignResourcePool VApp.Import VirtualMachine.Config.AddNewDisk
-
+openshift-datastore-level| vSphere Datastore | Datastore.AllocateSpace Datastore.Browse Datastore.FileManagement
```

Comment:
Can we add a `<br>` in between each privelege to make the table easier to read? Or would that make it more difficult for the user to create these roles? This has a couple benefits.
1) Makes the priveleges easier to read as each privelege is on its own line.
2) Makes the Privelege Set column narrower. This leaves more space for the other two columns, so that the role names and vSphere objects are not split across multiple lines.

**Original**
Role Name | vSphere object | Privelege Set
--- | --- | ---
openshift-datastore-level| vSphere Datastore | Datastore.AllocateSpace Datastore.Browse Datastore.FileManagement

**Proposed Replacement**
Role Name | vSphere object | Privelege Set
--- | --- | ---
openshift-datastore-level| vSphere Datastore | Datastore.AllocateSpace<br>Datastore.Browse<br>Datastore.FileManagement

Reply:
Sure. The `openshift-folder-level` might be interesting but will figure it out.